src/Bridges/bridge_optimizer.jl

```diff
@@ -1496,6 +1496,23 @@ function MOI.get(
     return unbridged_function(b, func)
 end
 
+function MOI.get(
+    b::AbstractBridgeOptimizer,
+    attr::MOI.ConstraintPrimal,
+    ci::MOI.ConstraintIndex,
+)
+    if is_bridged(b, ci)
+        MOI.throw_if_not_valid(b, ci)
+        return call_in_context(MOI.get, b, ci, attr)
+    elseif !Variable.has_bridges(Variable.bridges(b))
+        return MOI.get(b.model, attr, ci)
+    else
+        # Get the fallback, because the function might contain bridged
```

Comment:
I don't see why this would be an issue, let me take a look

Reply:
Take a look at https://github.com/jump-dev/MathOptInterface.jl/issues/2384#issuecomment-1880349485. Perhaps I'm missing something about how the substitutions should work.

Reply:
Thanks, that reduced example should help

Reply:
It's an issue for the same reason as 
https://github.com/jump-dev/MathOptInterface.jl/blob/79c19e09f024de009238a7b78af7cf355c62cc45/src/Bridges/bridge_optimizer.jl#L1380-L1403

Except that we can't call `unbridged_constraint_function` because the return from `MOI.get` is just a scalar, so we don't know what contributed to the primal and what was moved to the set.

Reply:
Ok, I see what's the issue. Because it's a scalar constraint, `normalize_and_add_constraint` moves the constant to the set so it does not contribute to `ConstraintPrimal` anymore. Could you add a comment explaining that this is due to this ? Then it's good to merge